api/v1/flow/flow.proto

```diff
@@ -236,7 +236,6 @@ enum TraceReason {
     REOPENED = 5 [deprecated=true];
     SRV6_ENCAP = 6;
     SRV6_DECAP = 7;
-    ENCRYPT_OVERLAY = 8;
```

Comment:
My understanding was that we can *never* drop stuff from the proto. But I'll let @kaworu speak to that :).